lib/BackgroundJob/ExpireGroupVersions.php

```diff
@@ -42,7 +50,42 @@ public function __construct(GroupVersionsExpireManager $expireManager, ITimeFact
 		$this->expireManager = $expireManager;
 	}
 
+	/**
+	 * Expiring groupfolder versions can be quite expensive.
+	 * We need to limit the amount of folders we expire per run.
+	 */
 	protected function run($argument) {
-		$this->expireManager->expireAll();
+		$lastFolder = $this->appConfig->getValueInt(Application::APP_ID, 'cron_last_folder_index', 0);
+		$folders = $this->folderManager->getAllFolders();
+
+		$folderCount = count($folders);
+		$currentRunHour = (int)date('G', $this->timeFactory->getTime());
```

Comment:
```suggestion
		$currentRunHour = (int)date('G', $this->time->getTime());
```
It’s named `$time` in `Job`.